### PR TITLE
Make rating_issue_profile_date a datetime.

### PR DESCRIPTION
**Pull request by shanear**
Well, this is awkward. Profile date is not a date. Saving it as a date in our database causes matching request issues to rating issues to break in the edit page because the profile dates don't match (one is a date, one is a datetime)

connects #7138

### CODE REVIEW

File: spec/feature/intake/edit_spec.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     Generators::Rating.build(
       participant_id: veteran.participant_id,
       promulgation_date: receipt_date + 1.day,
-      profile_date: receipt_date + 4.days,
+      profile_date: 24.days.ago,
```
Comment (shanear) [inline spec/feature/intake/edit_spec.rb]: This causes tests to fail before the migration.

File: db/schema.rb [hunk 2]
```diff
@@ -569,7 +569,7 @@
     t.string "review_request_type"
     t.bigint "review_request_id"
     t.string "rating_issue_reference_id"
-    t.date "rating_issue_profile_date"
+    t.datetime "rating_issue_profile_date"
```
Comment (jiahuang) [inline db/schema.rb]: Should there be a migration file in here? Thought the schema isn't used in upper envs

Comment (shanear) [reply db/schema.rb]: good call 😄 

i committed it

**Comment by leikkisa** [comment]
Oh, interesting, good catch.

**Pull request merged**